api/envoy/config/core/v3/protocol.proto

```diff
@@ -39,19 +39,16 @@ message QuicKeepAliveSettings {
   //
   // If zero, disable keepalive probing.
   // If absent, use the QUICHE default interval to probe.
-  google.protobuf.Duration max_interval = 1 [(validate.rules).duration = {
-    lte {}
-    gte {seconds: 1}
-  }];
+  google.protobuf.Duration max_interval = 1;
```

Comment:
should this also have `gte {nanos:1000000}`?

Reply:
No, I removed that constraint to be consistant with the comment.

Reply:
I see why we did this, b/c of this check: https://github.com/envoyproxy/envoy/blob/eff3ed3143d0b229e0ea5b2d65608c375d26afa5/source/common/quic/envoy_quic_client_session.cc#L269.